examples/tina-self-hosted-demo/pages/api/tina/[...routes].ts

```diff
@@ -0,0 +1,16 @@
+import { TinaNodeBackend } from '@tinacms/datalayer/dist/node'
+import { TinaAuthJSOptions, NextAuthAuthentication } from 'tinacms-authjs'
+
+import databaseClient from '../../../tina/__generated__/databaseClient'
+
+export const nextAuthOptions = TinaAuthJSOptions({
+  databaseClient: databaseClient,
+  secret: process.env.NEXTAUTH_SECRET,
+})
+
+const tinaNodeBackend = TinaNodeBackend({
+  authentication: NextAuthAuthentication({ nextAuthOptions }),
+  databaseClient,
+})
```

Comment:
can you update this to show this is setup running in dev mode (is local true/false)?

Reply:
Updated